python_apps/api_clients/api_clients/version2.py

```diff
@@ -45,9 +45,10 @@ def __init__(self, logger=None, config_path="/etc/airtime/airtime.conf"):
             self.logger.exception("Error loading config file: %s", config_path)
             sys.exit(1)
 
+    # TODO: Move this logic to api.
```

Comment:
This shouldn't be moved to the API. The formatting performed here is liquidsoap specific and ensures that it is formatted in the way the pypo-liquidsoap expects. If anything, this should be moved to pypo and out of the API client

Reply:
Ok, I guess we will have to think a bit about this in the future. I'll remove the comment for now.